eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -92,6 +92,8 @@ jobs:
         parameters:
           AgentImage: $(OsVmImage)
 
+      - template: /eng/pipelines/templates/steps/fix-linux-1es-configs.yml
```

Comment:
Nit, `1es` at a glance looks like `les`.  Suggestion `OneES` though if 1es is already everywhere then don't change it

Reply:
`OneES` won't match any search terms if someone is trying to figure out what this refers to, since they call it 1es themselves. I agree on the readability aspect though. Maybe just `fix-managed-image-apt-azure-sources` or something and explain the rest in the comments.